examples/flutter_gallery/lib/demo/material/cards_demo.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/foundation.dart' show required;
```

Comment:
Yeah, we do it with some things, but not with foundation.  Too much useful stuff in there.  :)